### PR TITLE
Added LiLo.Lite Xamarin.Forms app

### PR DESCRIPTION
**Pull request by GeorgeLeithead**
Added my personal project LiLo.Lite, which is a Xamarin.Forms app.

### CODE REVIEW

**Comment by InquisitorJax** [comment]
Hi,

Thanks for you submission. Sorry it's taken so long to get to this.

Did you check if the screenshots are rendering properly in the markdown?
This is what I'm seeing:
https://github.com/GeorgeLeithead/LiLo.Lite/blob/master/Screenshots/Phone_SplashScreen.png

please also follow the pattern of up to 4 images only - all aligned in a single row.

**Comment by GeorgeLeithead** [comment]
Images reduced to 4 and changed to use from Play store and sized to match others.

File: README.md [hunk 1]
```diff
@@ -124,3 +124,17 @@ A weekly round up of all the latest Xamarin news for developers.
...
+## LiLo.Lite
+A real-time cryptocurrency (crypto) price tracking application. It provides a quick and easy way to watch the top crypto currency information and charts. With extremely low battery usage, you can leave the application running in the background and have crypto information available right at your fingertips.
+
+##### Links
+- [WebSite](https://www.internetwideworld.com/lilolite)
+- [Google Play Store](https://play.google.com/store/apps/details?id=com.internetwideworld.lilo.lite&pcampaignid=github)
+- Made by [George Leithead](https://twitter.com/GeorgeLeithead) support account [LiLo](https://twitter.com/LiLoMobileApp)
+
+##### Video / Screenshots
+![Splash Screen](https://play-lh.googleusercontent.com/UocetXUITtX_x4ebU4ziUH2cCeFfehKBILhKypk43YsCvZWATy87MpEzj-VzHGzR7bS-=w720-h310-rw)
```
Comment (GeorgeLeithead) [inline README.md:137]: Changed to use sized image from play store.

File: README.md [hunk 2]
```diff
@@ -124,3 +124,17 @@ A weekly round up of all the latest Xamarin news for developers.
...
+![Home Page - Light theme - Portrait](https://play-lh.googleusercontent.com/gmeMUeQB85kTCdMRs4DbVBH6vKRcD-k5HlA37xLHV_xyXKuZp2mXiSogT-F54pmLePE=w720-h310-rw)
```
Comment (GeorgeLeithead) [inline README.md:138]: Changed to use sized image from play store.

File: README.md [hunk 3]
```diff
@@ -124,3 +124,17 @@ A weekly round up of all the latest Xamarin news for developers.
...
+![Chart page - Light theme](https://play-lh.googleusercontent.com/6fKspX1MRmLBIUuR-Sx9LMj30vmJQzPYoKo6lFa9oyvsBtev1f4bo84SpW8d22OVosU=w720-h310-rw)
```
Comment (GeorgeLeithead) [inline README.md:139]: Changed to use sized image from play store.

File: README.md [hunk 4]
```diff
@@ -124,3 +124,17 @@ A weekly round up of all the latest Xamarin news for developers.
...
+![Settings page - Light theme](https://play-lh.googleusercontent.com/yokvptXYZkwWGiuw_QoKNpJ7VYBSWeRg_uu29E71BczPRH6FWjYDTGrzPFh7VdbtLA=w720-h310-rw)
```
Comment (GeorgeLeithead) [inline README.md:140]: Changed to use sized image from play store.

**Review by GeorgeLeithead — COMMENTED** [review]
Reduced to only using 4 images, sibe-by-side.

**Pull request merged**